PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cff.py

```diff
@@ -18,3 +18,25 @@
     patJets
     )
 makePatJets = cms.Sequence(makePatJetsTask)
+
+from RecoBTag.ImpactParameter.pfImpactParameterTagInfos_cfi import * #pfImpactParameterTagInfos
+from RecoBTag.SecondaryVertex.pfSecondaryVertexTagInfos_cfi import * #pfSecondaryVertexTagInfos
+from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderTagInfos_cfi import * #pfInclusiveSecondaryVertexFinderTagInfos
+from RecoBTag.Combined.deepFlavour_cff import * #pfDeepFlavourTask
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+
+#make a copy to avoid labels and substitution problems
+_makePatJetsCopy = makePatJets.copy()
+
+run2_miniAOD_80XLegacy.toReplaceWith(
+    makePatJets, 
+    cms.Sequence(
```

Comment:
I actually thought that this PR comes with an operational workflow at hand.

I would expect that something like the following should work
```
cmsDriver.py step3  --conditions auto:run2_mc --process PAT -n 10 --era Run2_2016,run2_miniAOD_80XLegacy --eventcontent MINIAODSIM --runUnscheduled  -s PAT --datatier MINIAODSIM  -n -1 --filein file:aod.root  --fileout file:step3.root```

In any case, to be complete, changes in this file should be applied to makePatJetsTask. 
And my expectations are that the modifier to makePatJetsTask will make the makePatJets sequence change automatically.